test/Interop/Cxx/namespace/Inputs/namespaces.h

```diff
@@ -0,0 +1,72 @@
+namespace Space {
+namespace Ship {
+void test1() {}
+void test4();
+void test5();
```

Comment:
Could you add tests that show that the mangled names of this namespaces refer to the `__ObjC` module?

For example, add an `extension Space { func swiftFuncInCxxNamespace() {} }` and check the name that IRGen gives it.

Reply:
Done. (Need to fix the tests for Windows, though.)